docs/nva-publication-channels.yaml

```diff
@@ -22,6 +22,62 @@ externalDocs:
 servers:
   - url: https://api.nva.unit.no/channels
 paths:
+  /journal:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+            Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${PublicationChannelsHandler.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+      - Journal
+      summary: Find journal by ID
```

Comment:
Journal by ID? You're defining journal search by the look of the rest of the template.

Reply:
...Find journal... Returns a list of journals matching criteria